tools/remove-clangcldocker.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+tuser=$(echo $USER | tr -dc 'a-z')
```

Comment:
Might want to add `set -e` to fail the script if any line fails.

Reply:
Done.